dts/bindings/clock/st,stm32h7-lse-clock.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022, SILA Embedded Solutions GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 LSE Clock
+
+compatible: "st,stm32h7-lse-clock"
+
+include: [fixed-clock.yaml]
+
+properties:
+    driving-capability:
+      type: int
+      required: true
+      description: |
+        LSE driving capability
+      enum:
+        - 0 # LSEDRV = 0x0, lowest
+        - 1 # LSEDRV = 0x1, medium low
+        - 2 # LSEDRV = 0x2, medium high
+        - 3 # LSEDRV = 0x3, highest
```

Comment:
It would be nicer if you could document the enum values in the description, so that they would show up in the bindings index page which will be created automatically for this YAML file under https://docs.zephyrproject.org/latest/build/dts/api/bindings.html.

Reply:
Thought about the exact same thing already, then forgot about it again. Thanks for the reminder!